### DISCUSSION TITLE
V6 multilevel menu items

### DISCUSSION THREAD

**Discussion by froid24** [Laravel-Backpack/community-forum#614] [Q&A (Help)]
Hi guys,

I saw a lot of talk about having multiple levels for the menu in previous versions (where a submenu option can have a submenu of its own). Where can I find an example of how it is done in V6 (Tabler)?

Also, at some point menu options not having any suboptions were automatically hidden. is this not the case anymore? Because that was really nice.

Thank you!

**Comment by karandatwani92** [comment]
Hey @pxpm for the Tabler theme, I took the snippet from the following demos.
https://preview.tabler.io
https://preview.tabler.io/layout-vertical.html

They are using a `dropend` class under the `Interface->Authentication` menu item **but it's not working as expected.** 
![Screenshot 2023-08-07 at 2 39 00 PM](https://github.com/Laravel-Backpack/community-forum/assets/8214221/45ea28f0-1821-47e7-9c94-61f21be2eb17)

**Comment by karandatwani92** [comment]
Hey @froid24, Thanks for reporting the issue. We will be fixing it soon. https://github.com/Laravel-Backpack/CRUD/issues/5268

Sorry for inconvenience, Good day
